### PR TITLE
[feat] Improved accessibility of Message object, attach more constants to client

### PR DESCRIPTION
**Pull request by brendacs**
### Purpose of pull request

**Related Issue**: resolves N/A

Improved accessibility of Message object, attach more constants to client to clean up

### Pull request checklist

- [x] Branch and PR are named correctly
- [ ] Updated relevant documentation
- [x] Tested with a tester bot
- [ ] Wrote unit tests for changes

### Testing instructions

Ping bot should still work

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -13,12 +13,4 @@ export const envs = {
...
-export const validCmds = [
-  'ama',
-  'faq',
-  'pins',
-  'ping',
-  'salary',
-  'settings',
-  'perms'
-];
+export const validCmds = ['ama', 'faq', 'pins', 'ping', 'salary', 'settings'];
```
Comment (sirNikolai) [inline src/constants.js:16]: we should probably normalize when to break up into separate lines.

File: src/utils/dispatcher.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
 import client from '../client';
-import { prefix, validCmds } from '../constants';
 
 export const parseMessage = (msg) => {
+  client.msg = msg;
```
Comment (sirNikolai) [inline src/utils/dispatcher.js:4]: very javascripty

**Review by sirNikolai — APPROVED** [review]

File: src/constants.js (hunk 1 above)
Comment (brendacs) [reply src/constants.js]: It's normalized - based on eslint + prettier

File: src/utils/dispatcher.js (hunk 2 above)
Comment (brendacs) [reply src/utils/dispatcher.js]: oh yes. :)

**Pull request merged**